packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -102,7 +104,96 @@ exports.sourceNodes = async (
     return
   }
 
-  const drupalFetchActivity = reporter.activityTimer(`Fetch data from Drupal`)
+  fastBuilds = fastBuilds || false
+  if (fastBuilds) {
```

Comment:
Nit: this block makes a rather long function even longer. Maybe this could be a separate function?

Reply:
I agree with this and have planned to rework this entire function a bit to separate out the major parts (webhook handling, fastbuilds, and full node sourcing) into smaller utility functions. I think this can be left as-is for now and I will revisit a larger refactoring during the next phase of dev.